azure-page/src/App.js

```diff
@@ -0,0 +1,48 @@
+import logo from "./logo.svg";
+import img from "./manage.png";
+import "./index.css";
+// import Dropdown from "./components/dropdown/explore/explore";
+import Header from "./components/header";
+import Banner from "./components/banner";
+import Card from "./components/Card";
+// import Explore from "./components/dropdown/explore/explore";
+// import DropDown from "./components/dropdown/explore/DropdownMain";
```

Comment:
No commented codes for review. Remove all, o/w mention why its kept commented.

Reply:
ok.Resolved